src/tools/illink/src/linker/Linker/Driver.cs

```diff
@@ -83,6 +83,9 @@ public static int Main (string[] args)
 				Debug.Assert (context != null);
 				return context;
 			}
+			set {
+				context = value;
```

Comment:
Could you please add an assert that the context was `null` before this call? To make sure it's only ever set once.

Reply:
Done.